doc/gtk-3.0.css

```diff
@@ -19,78 +19,254 @@
   the widget type can be configured unless they are named in code.
 */
 
-/* Register sheet font setting */
-.sheet {
-  font: 18px arial, sans-serif;
-}
 
-.sheet calendar {
+/* Application wide font setting */
+* {
   font: 12px arial, sans-serif;
 }
 
 
-/* Register User Colors */
-.header-color {
+/* Scollbar size */
+scrollbar slider {
+    /* Size of the slider */
+    min-width: 20px;
+    min-height: 20px;
+    border-radius: 22px;
+
+    /* Padding around the slider */
+    border: 5px solid transparent;
+}
+
+
+/* Register sheet font setting, use .gnc-class-sheet < gtk3.20 */
```

Comment:
As we now require Gtk 3.22, I think all references to use classes for older gtk versions like the above can be removed.

Reply:
OK will have a look, that also means getting rid of gnucash-310.css, will sort this out during week